lib/cocoapods-core/gem_version.rb

```diff
@@ -1,5 +1,5 @@
 module Pod
   # The version of the cocoapods-core.
   #
-  CORE_VERSION = '1.10.0.rc.1'.freeze unless defined? Pod::CORE_VERSION
+  CORE_VERSION = '1.10.0.rc.2'.freeze unless defined? Pod::CORE_VERSION
```

Comment:
this is bumped as part of the release process, no need to do it manually I will take care of it once we ship next version of 1.10.

Reply:
Thanks!